File: soaint-sgd-enterprise-artifacts/canonical-api-contract/src/main/java/co/com/soaint/foundation/canonical/correspondencia/ProyectorDTO.java
package co.com.soaint.foundation.canonical.correspondencia;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.xml.bind.annotation.XmlRootElement;
import java.io.Serializable;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder(builderMethodName = "newInstance")
@XmlRootElement(namespace = "http://soaint.com/domain-artifacts/proyector/1.0.0")
public class ProyectorDTO implements Serializable {

    private static final long serialVersionUID = 1L;
    private ConstanteDTO sedeAdministrativa;
    private DependenciaDTO dependencia;
    private FuncionarioDTO funcionarioProyector;
    private ConstanteDTO tipoPlantilla;

}

File: soaint-sgd-ui/soaint-sgd-web/src/app/domain/ProyectorDTO.ts
import {DependenciaDTO} from './dependenciaDTO';
import {FuncionarioDTO} from './funcionarioDTO';
import {ConstanteDTO} from './constanteDTO';

export interface ProyectorDTO {
  sede: ConstanteDTO;
  dependencia: DependenciaDTO;
  funcionario: FuncionarioDTO;
  tipoPlantilla: ConstanteDTO;
}
